### PR TITLE
31 unit tests location and hall

### PR DESCRIPTION
**Pull request by stephensmeal**
Requesting peer review of the following class unit tests to be merged into the integ branch:
Location - test_location.py
Hall - test_hall.py

### CODE REVIEW

File: test/unit/test_hall.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+def test_init():
+    assert type(test_hall1) is Hall
+
+
+def test_create_neighbors():
+    if test_hall1.is_neighbor(test_hall2) is False:
+        assert True
+    else:
+        assert False
+    test_hall1.create_neighbors([test_hall2, test_hall3])
```
Comment (patmay) [inline test/unit/test_hall.py]: lines 17-20 would be better written as:
assert not test_hall1.is_neighbor(test_hall2)

**Review by patmay — DISMISSED** [review]
Tests are very thorough and all pass. I suggest you refactor the assertion statements that take the form "if (condition) then assert true else assert false".

**Comment by stephensmeal** [comment]
Addressed comments... refactored Assertions for better readability.

**Comment by patmay** [comment]
much better. all tests pass.

**Review by patmay — APPROVED** [review]

**Review by yo09975 — APPROVED** [review]

**Pull request merged**